Description: IdentiCurse checks for config.json in
 /usr/lib/python*/*-packages/identicurse/, change this behaviour to make it
 check it in /usr/share/identicurse/.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=621895
Author: Alessio Treglia <alessio@debian.org>
Modified: Kacper Kowalik <xarthisius@gentoo.org>

--- a/src/identicurse/identicurse.py
+++ b/src/identicurse/identicurse.py
@@ -133,8 +133,8 @@ class IdentiCurse(object):
             else:
                 import getpass, time
                 # no config yet, so let's build one
-                print os.path.join(self.path, "config.json")
-                config.config.load(os.path.join(self.path, "config.json"))
+                print os.path.join("/", "usr", "share", "identicurse", "config.json")
+                config.config.load(os.path.join("/", "usr", "share", "identicurse", "config.json"))
                 print "No config was found, so we will now run through a few quick questions to set up a basic config for you (which will be saved as %s so you can manually edit it later). If the default (where defaults are available, they're stated in []) is already fine for any question, just press Enter without typing anything, and the default will be used." % (config.config.filename)
                 print "This version of IdentiCurse supports OAuth login. Using OAuth to log in means that you do not need to enter your username and password."
                 use_oauth = raw_input("Use OAuth [Y/n]? ").upper()
--- a/setup.py
+++ b/setup.py
@@ -36,7 +36,7 @@
 
     license="GPLv3+",
 
-    data_files=[('identicurse',['README', 'conf/config.json'])],
+    data_files=[('share/identicurse',['README','conf/config.json'])],
     packages=find_packages('src'),
     package_dir={'': 'src'},
     include_package_data=True,
